nimbus/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
-name = "experiments"
+name = "nimbus-sdk"
```

Comment:
I chose this, just in case we wanted to put a crate up on crates.io as this name wasn't taken (nimbus was).